04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,63 @@
+const Product = require('../models/product')
+
+const getAllProductsStatic = async (req, res) => {
+   
+    //throw new Error('testing async errors')
+    const products = await Product.find({ price: { $gt: 30 } })
+    .sort('price')
+    .select('name price')
+    //.limit(10).skip(1)
+    res.status(200).json({ products, nbHits:products.length})
+
+}
```

Comment:
it always good to implement error handling when working with asynchronous operation 
> Hint: A try-catch block will do the work